services/graph/apps/worker/src/graph_publisher/graph.publisher.processor.service.ts

```diff
@@ -36,11 +47,11 @@ export class GraphUpdatePublisherService extends BaseConsumer implements OnAppli
   constructor(
     @InjectRedis() private cacheManager: Redis,
     @InjectQueue(QueueConstants.GRAPH_CHANGE_PUBLISH_QUEUE) private graphChangePublishQueue: Queue,
-    @InjectQueue(QueueConstants.GRAPH_CHANGE_NOTIFY_QUEUE) private graphChangeNotifyQueue: Queue,
```

Comment:
nit: maybe add a post-v1 todo to update the architecture if a queue was removed

Reply:
Actually, the architecture diagram just has a box that says "Graph Notifier"; it doesn't say that it's a queue. Less detail FTW 😅